src/parenthetics.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+
+def paren_check(string):
+    count = 0
+    for idx, l in enumerate(string):
```

Comment:
I am curious why you are doing it this way? You don't actually use the idx..
```
for l in string:
``` would provide everything you are using..